tests/unit/models/360-core.js

```diff
@@ -165,6 +181,14 @@ describe("SerSol 360 Core Model >", function() {
       expect(data).toBeDefined();
       expect(serSol360Core.getBrowZineEnabled(data)).toEqual(true);
     });
+
+    it("should return null when the BrowZine API response for a journal is empty", function() {
+      var data = serSol360Core.getData({
+        "data": []
+      });
+      expect(data).toBeUndefined();
+      expect(serSol360Core.getBrowZineEnabled(data)).toEqual(null);
```

Comment:
_non-blocking PR comment_

I'm curious if we'd think `getBrowZineEnabled()` should always return a boolean, or if `null` is desired for some reason, maybe an Angular-specific reason.  

If there's no data, seems like `getBrowZineEnabled()` returning false would be fine, since as far as we know it's not enabled... but perhaps it's even more appropriate to send `null`, because it's neither `true` nor `false`, we just simply do not know since we got no data.

I think this comment is mostly me thinking out loud about what `null` signifies, so I think no change needed here 😊

Reply:
> perhaps it's even more appropriate to send null, because it's neither true nor false, we just simply do not know since we got no data

This is my thought, couldn't have said it better.